win32.mak

```diff
@@ -11,11 +11,11 @@ auto-tester-build:
 	cd ..
 
 auto-tester-test:
-	cd test
-	$(MAKE)
-	cd ..
-	cd samples
-	gmake -f win32.mak DMD=..\src\dmd.exe MODEL=$(MODEL) "LIB=..\..\phobos;$(LIB)" \
-		"DFLAGS=-I..\..\druntime\import -I..\..\phobos -m$(MODEL)"
-	cd ..
-
+	echo "Windows builds have been disabled"
+	#cd test
+	#$(MAKE)
+	#cd ..
+	#cd samples
+	#gmake -f win32.mak DMD=..\src\dmd.exe MODEL=$(MODEL) "LIB=..\..\phobos;$(LIB)" \
+	#	"DFLAGS=-I..\..\druntime\import -I..\..\phobos -m$(MODEL)"
```

Comment:
Should we
- integrate this into `run.d`
- hack into one of Azure pipelines
- drop the samples (they are super old)

Reply:
Probably best to drop them or move them into druntime/phobs (they primarily "test" libraries AFAICT)